.github/workflows/generate-docs.yml

```diff
@@ -1,54 +1,47 @@
-name: Generate Docs
+name: Update Docs
 
-on: 
+on:
   workflow_dispatch:
 
 jobs:
-  generate-docs:
+  update-docs:
     runs-on: ubuntu-latest
 
     steps:
-    - name: Checkout repository
-      uses: actions/checkout@v2
+    - name: Check out code
+      uses: actions/checkout@v3
 
-    - name: Setup Go
-      uses: actions/setup-go@v4
+    - name: Set up Go
+      uses: actions/setup-go@v2
       with:
-        go-version: 1.20
+        go-version: 1.18 
```

Comment:
What is the reason we need to use outdated go version for this?

Reply:
tbh, I have no clue, somehow using 1.20 with v4 of github actions breaks my script for {unknown}{unknown} reasons. If you think using recent go should be the move, I can look into what the problem is and try using 1.20 with v4

Reply:
I'd file this into an issue, and handle later